src/private/rapidxml_formatter.cpp

```diff
@@ -1,10 +1,31 @@
 #include "adm/private/rapidxml_formatter.hpp"
 #include "adm/private/rapidxml_wrapper.hpp"
+#include "adm/serial/frame_format_id.hpp"
+#include <adm/serial/changed_ids.hpp>
+#include "adm/private/changed_id_traits.hpp"
 
 namespace adm {
   namespace xml {
 
     namespace detail {
+      template <typename T, typename F>
+      auto wrapWithTimeRef(F fn, TimeReference timeReference) {
+        return [fn, timeReference](XmlNode &node, T const &element) {
+          fn(node, element, timeReference);
+        };
```

Comment:
could make this a generic lambda (since c++14 is used) to avoid having to specify T when using it?

Reply:
700e2d6e590ff9a7aca9c460644d44895d4c4e1d